lab-geoff/test/test.js

```diff
@@ -0,0 +1,12 @@
+var assert = require('assert');
+var greet = require('../greet.js');
+
+describe('greet.js', function() {
+  it('greet should not be an empty object', function() {
```

Comment:
These are great tests, but you're not testing that the greet function actually returns the expected string. 😄